docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  vector:
+    image: actian/vector:community
```

Comment:
Does this imply that we have the image pushed to our private Actian docker registry? If so, it should go into documentation. Another question: I thought, we do not offer community edition anymore. So this should then be removed, right?

Reply:
This file is meant to be an example mostly and no, there was no intention to use hub.docker.com at this time.  Since this project builds the image, it will be local to the users environment.  Personally I think it would be cool to have a Vector image publicly available on Docker Hub if we could deal with the licensing issues.

In any case, :community is a bug because the Dockerfile was modified to remove community references and I neglected to remove it here after the merge.